charts/fulcio/values.yaml

```diff
@@ -12,8 +12,8 @@ server:
     registry: gcr.io
     repository: projectsigstore/fulcio
     pullPolicy: IfNotPresent
-    # crane digest gcr.io/projectsigstore/fulcio:v0.5.3
-    version: "sha256:61081295a8f75ed7537b5d1f8c7320e078dc00e4562c0bf605fbefa062c690de"
+    # -- v0.5.3
```

Comment:
The crane command is now documented in CONTRIBUTING.md see #300

Reply:
I would strongly prefer to leave this comment in the file as well, I don't think it's likely someone doing a release will read CONTRIBUTING.md before starting